terraform/service_cleanup_export.tf

```diff
@@ -79,7 +79,7 @@ resource "google_cloud_run_service" "cleanup-export" {
 
         resources {
           limits = {
-            cpu    = "2"
+            cpu    = "2000m"
```

Comment:
Not sure if this is a recent Cloud Run API change or a Terraform bug, but I was getting a consistent diff on this.